install

```diff
@@ -738,10 +744,24 @@ if [ "$ignore_dependencies" = "f" ]; then
         if [ "$in_place" = "t" ]; then
             loudCmd "DEBIAN_FRONTEND=noninteractive apt-get -y -m install git"
         fi
+
+        # Ubuntu 20.04 Focal Fossa is missing some dependencies
+        # https://github.com/LibreTime/libretime/issues/1109
+        if [[ "$is_ubuntu_focal" -eq "true" ]]; then
+            echo
+            echo "ERROR: This distribution probably does not package Liquidsoap."
```

Comment:
Ubuntu Focal does package Liquidsoap: https://packages.ubuntu.com/focal/liquidsoap

Reply:
Sadly, Focal does not appear to include these in its repo:

```
liquidsoap-plugin-alsa
liquidsoap-plugin-ao
liquidsoap-plugin-faad
liquidsoap-plugin-flac
liquidsoap-plugin-icecast
liquidsoap-plugin-lame
liquidsoap-plugin-mad
liquidsoap-plugin-ogg
liquidsoap-plugin-portaudio
liquidsoap-plugin-pulseaudio
liquidsoap-plugin-taglib
liquidsoap-plugin-voaacenc
liquidsoap-plugin-vorbis
```

Reply:
```
sudo apt install liquidsoap-plugin-vorbis
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package liquidsoap-plugin-vorbis is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  liquidsoap

E: Package 'liquidsoap-plugin-vorbis' has no installation candidate
```

Does this mean we no longer need the extra plugins?

Reply:
Yes. The plugins are no longer separate packages